common/reviews/api/rush-lib.api.md

```diff
@@ -1300,6 +1320,18 @@ export class RushUserConfiguration {
     static initializeAsync(): Promise<RushUserConfiguration>;
 }
 
+// @beta
+export class SubspaceConfiguration {
+    readonly availableSubspaceSet: Set<string>;
+    // @internal
+    readonly configuration: Readonly<ISubspaceConfigurationJson>;
```

Comment:
Is exposing the raw JSON file useful?

Reply:
+1 Let's avoid that unless TikTok specifically needs it for something

Reply:
Changed configuration to a private property